bin/contentful_bootstrap

```diff
@@ -75,9 +75,10 @@ if subcommands.has_key? command
   end
 
   if space.nil? || space.empty?
-    puts "Usage: contentful_bootstrap <command> <space> [options]"
+    ARGV << '-h'
+    subcommands[command].order!
     puts
-    puts "You must provide a valid space name. Exiting!"
+    puts "You must provide a valid space. Exiting!"
```

Comment:
Maybe we can change this to "No arguments specified" or sth? Depending on the command it needs to be a an ID or a new name and now the message doesn't really help in either case.